neuralmonkey/encoders/cnn_encoder.py

```diff
@@ -202,28 +204,30 @@ def rnn_encoder(inputs, last_layer_size, scope):
             encoder_state = rnn_encoder(
                 encoder_ins, last_layer_size, "encoder-forward")
 
+            # pylint: disable=redefined-variable-type
             if bidirectional:
                 backward_encoder_state = rnn_encoder(
                     list(reversed(encoder_ins)),
-                    last_layer_size,
-                    "encoder-backward")
-                # pylint: disable=redefined-variable-type
+                    last_layer_size, "encoder-backward")
                 encoder_state = tf.concat(
                     1, [encoder_state, backward_encoder_state])
 
             self.encoded = encoder_state
 
-            self.attention_tensor = \
+            self.__attention_tensor = \
```

Comment:
dvě podtržítka? Neni to moc?
plus, backslash jde vyrefakorovat do konce řádky za `reshape(`